src/main/java/team05/integrated_feed_backend/module/post/controller/PostStatisticsController.java

```diff
@@ -0,0 +1,33 @@
+package team05.integrated_feed_backend.module.post.controller;
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+import team05.integrated_feed_backend.common.BaseApiResponse;
+import team05.integrated_feed_backend.exception.code.StatusCode;
+import team05.integrated_feed_backend.module.post.dto.request.PostStatisticsListReq;
+import team05.integrated_feed_backend.module.post.dto.response.PostStatisticsListRes;
+import team05.integrated_feed_backend.module.post.service.PostStatisticsService;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/posts")
+public class PostStatisticsController {
+
+	private final PostStatisticsService postStatisticsService;
+
+	@GetMapping("/statistics")
+	public BaseApiResponse<List<PostStatisticsListRes>> getPostStatistics(
+		@RequestParam(defaultValue = "date") String type, @RequestParam(required = false) String hashtag,
+		@RequestParam(required = false, defaultValue = "count") String value,
+		@RequestParam(required = false) String start, @RequestParam(required = false) String end) {
```

Comment:
```suggestion
@RequestParam(required = false)
@DateTimeFormat(pattern = "yyyy-MM-dd")
LocalDateTime start, 
@RequestParam(required = false) 
@DateTimeFormat(pattern = "yyyy-MM-dd")
LocalDateTime end) {
```
현재 방법도 너무 좋구용! 제가 알고있는 방법을 하나 공유하자면 요렇게 `@DateTimeFormat` 애너테이션을 붙이면 바로 `LocalDateTime`으로 받아올수도 있습니당!

Reply:
우와.. 의진님 최고..👍 덕분에 `@DateTimeFormat` 사용해서 `LocalDate`로 받아서 `LocalDatetime`으로 변환했습니다!!
꿀팁 감사합니다😉